exercise-app/src/test/java/org/exercise/app/AppControllerTest.java

```diff
@@ -0,0 +1,88 @@
+package org.exercise.app;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+import java.nio.charset.StandardCharsets;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import com.examples.attsw.exercise.core.controller.IEmployeeController;
+import com.examples.attsw.exercise.core.model.Employee;
+
+public class AppControllerTest {
+
+	private AppController appController;
```

Comment:
You could use the alternative mechanism to initialize and assign mocks, i.e., InjectMocks and Mock annotations and then initMocks in the setup (see the tutorial)

Reply:
I know but i think this is more readable then the other mechanism... anyway i'm going to use InjectMocks.